Authorization/Authorization/en.lproj/Localizable.strings

```diff
@@ -14,7 +14,10 @@
 "SIGN_IN.FORGOT_PASS_BTN" = "Forgot password?";
 "SIGN_IN.AGREEMENT" = "By signing in to this app, you agree to the [%@ End User License Agreement](%@) and [%@ Terms of Service and Honor Code](%@) and you acknowledge that %@ and each Member process your personal data in
 accordance with the [Privacy Policy.](%@)";
-
+"SIGN_IN.SSO_HEADING" = "Start today to build your career with confidence";
+"SIGN_IN.SSO_SUPPORTING_TEXT" = "An integrated set of knowledge and empowerment programs to develop the components of the endowment sector and its workers";
+"SIGN_IN.SSO_LOG_IN_TITLE" = "Sign IN";
```

Comment:
To double-check, is it intentionally used as an uppercase in "Sign IN," or should it be "Sign In" or "Sign in"?
Thank you.

Reply:
No, it is not intentional. It should be ‘Sign in’ (with only the first letter capitalized). 
Thank you for pointing that out!